.github/workflows/ci.yml

```
@@ -10,8 +10,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04, macos-11, windows-2019]
-        sofa_branch: [master]
+        os: [ubuntu-22.04, macos-12, windows-2022]
+        sofa_branch: [v23.12]
```

```suggestion
        sofa_branch: [master]
```

The master branch should be compiled against master. But I will propagate your change in the 23.12 branch